addons/website_event_track/views/event_track_templates.xml

```diff
@@ -441,8 +447,8 @@
         <li><b>Proposed By</b>: <t t-esc="track.partner_name or track.partner_email"/></li>
         <li><b>Mail</b>: <a t-attf-href="mailto:#{track.partner_email}"><t t-esc="track.partner_email"/></a></li>
         <li><b>Phone</b>: <t t-esc="track.partner_phone"/></li>
-        <li><b>Speakers Biography</b>: <t t-esc="track.partner_biography"/></li>
-        <li><b>Talk Introduction</b>: <t t-esc="track.description"/></li>
+        <li><b>Speakers Biography</b>: <t t-raw="track.partner_biography"/></li>
+        <li><b>Talk Introduction</b>: <t t-raw="track.description"/></li>
```

Comment:
Be extra careful with "t-raw", even though I think it's ok for this case.

Reply:
Why using t-raw ? isn't it possible to use t-field ?